### ISSUE TITLE
Update import statement

### ISSUE THREAD

**Issue by MrVa55** [Web3Auth/web3auth-wagmi-connector#55]
In node_modules/@web3auth/web3auth-wagmi-connector/dist/ in the file web3authWagmiConnector.esm.js

Currently reads:
import { getAddress } from 'ethers/lib/utils';

But this throws an error that is resolved by changing it to:
import { getAddress } from 'ethers/lib/utils.js';

**Comment by ander-db** [comment]
I got the same error:
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'project/node_modules/ethers/lib/utils' imported from project/node_modules/@web3auth/web3auth-wagmi-connector/dist/web3authWagmiConnector.esm.js
Did you mean to import ethers/lib/utils.js?
    at new NodeError (node:internal/errors:399:5)
    at finalizeResolution (node:internal/modules/esm/resolve:326:11)
    at moduleResolve (node:internal/modules/esm/resolve:945:10)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  page: '/'
}
```

**Comment by vpavlin** [comment]
After modifying the import to add the `.js`, did you also see the following error?

```
SyntaxError: Named export 'ADAPTER_STATUS' not found. The requested module '@web3auth/base' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@web3auth/base';
const { ADAPTER_STATUS, WALLET_ADAPTERS, CHAIN_NAMESPACES } = pkg;
```

When I follow the suggestion and import the default export from `base`, I get another error:

```
TypeError: Cannot destructure property 'ADAPTER_STATUS' of '_web3auth_base__WEBPACK_IMPORTED_MODULE_1__.default' as it is undefined.
```

So right now I am stuck. Any suggestions/pointers would be highly appreciated.

**Comment by MrVa55** [comment]
Yes, I got that error too, but resolved it by replacing 

import { ADAPTER_STATUS, WALLET_ADAPTERS, CHAIN_NAMESPACES } from '@web3auth/base';

with 

import pkg from '@web3auth/base';
const { ADAPTER_STATUS, WALLET_ADAPTERS, CHAIN_NAMESPACES } = pkg;

as the error message suggested. After that it was working as intended.

**Comment by vpavlin** [comment]
Hmm..can you share your package.json, so that I see the versions of wagmi and web3auth packages you have? I have upgraded mine in the process of trying to fix it, so maybe that is why I get different errors

**Comment by MrVa55** [comment]
"@web3auth/base": "^5.0.1",
    "@web3auth/modal": "^5.0.1",
    "@web3auth/openlogin-adapter": "^5.0.1",
    "@web3auth/web3auth-wagmi-connector": "^3.0.0",
    "wagmi": "^0.12.2"

**Comment by recidive** [comment]
Hello, is there a fix or a workaround for this that doesn't entail modifying files in `node_modules`? Thanks!

**Comment by ander-db** [comment]
> Hello, is there a fix or a workaround for this that doesn't entail modifying files in `node_modules`? Thanks!

Edit: I've noticed that my approach was already on the thread by @MrVa55 . That solution worked for me

**Comment by yashovardhan** [comment]
Hey all, 

Can you please check if this issue was resolved for you all in `3.0.1` release?

**Comment by ander-db** [comment]
> Hey all,
> 
> Can you please check if this issue was resolved for you all in `3.0.1` release?

Version: `3.0.1`

Got this error:
```
file:///home/anderdb/project/node_modules/@web3auth/web3auth-wagmi-connector/dist/web3authWagmiConnector.esm.js:3
import { ADAPTER_STATUS, WALLET_ADAPTERS, CHAIN_NAMESPACES } from '@web3auth/base';
         ^^^^^^^^^^^^^^
SyntaxError: Named export 'ADAPTER_STATUS' not found. The requested module '@web3auth/base' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@web3auth/base';
const { ADAPTER_STATUS, WALLET_ADAPTERS, CHAIN_NAMESPACES } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5) {
  page: '/'
}
```

Modifying the file as the server error suggest fix the error.

**Comment by D-lyw** [comment]
> Hey all,
> 
> Can you please check if this issue was resolved for you all in `3.0.1` release?

Still error in `3.0.1`

```shell
file:///Users/d-lyw/Work/FirstPlay/node_modules/@web3auth/web3auth-wagmi-connector/dist/web3authWagmiConnector.esm.js:3
import { ADAPTER_STATUS, WALLET_ADAPTERS, CHAIN_NAMESPACES } from '@web3auth/base';
         ^^^^^^^^^^^^^^
SyntaxError: Named export 'ADAPTER_STATUS' not found. The requested module '@web3auth/base' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@web3auth/base';
const { ADAPTER_STATUS, WALLET_ADAPTERS, CHAIN_NAMESPACES } = pkg;
```

**Comment by recidive** [comment]
Still same error for me too in `3.0.1`.

**Comment by recidive** [comment]
Can we raise the priority of this issue? This looks critical to me as I'm unable to launch a project. Are there any alternatives other than having to use an old, buggy, version?

**Comment by chaitanyapotti** [comment]
Fixed in 3.0.2 and released. Thanks
